### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by michaelnakonechny**
add styles
add remove button in shopping cart
add header footer layout components

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -30,12 +32,21 @@ const defaultIngredients = [
 function MyApp({ Component, pageProps }) {
   const [ingredients, setIngredients] = useState(defaultIngredients);
 
+  // this renders on "localhost:3000" -> index.js
+  // and on "localhost:3000/shoppingcart" -> shoppingcart.js
+  // and on ".../whatever" -> whatever.js
+  // it always gives the rendered js file two properties: "ingredients" and "onUpdateIngredients"
```
Comment (ThomasKirst) [inline pages/_app.js]: Please remove comments

File: pages/_app.js [hunk 2]
```diff
@@ -30,12 +32,21 @@ const defaultIngredients = [
...
+    <>
+      <Header />
+      <Component
+        {...pageProps}
+        ingredients={ingredients}
+        onUpdateIngredients={setIngredients}
+      />
+
+      <Footer />
+    </>
```
Comment (ThomasKirst) [inline pages/_app.js]:
```suggestion
    <Layout>
      <Component
        {...pageProps}
        ingredients={ingredients}
        onUpdateIngredients={setIngredients}
      />
    </Layout>
```

Comment (ThomasKirst) [reply pages/_app.js]: Your `Layout` component should contain the `Header` and `Footer`.

File: pages/about.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+function About() {
+  return (
+    <>
+      <h1>This page was made with a lot of pain</h1>
```
Comment (ThomasKirst) [inline pages/about.js]: 😅

File: pages/components/Footer.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+
+export default function Footer() {
+  return (
+    <NavFooter>
+      <Navbar>
+        <Link href="/">
+          <Navlinks>HOME</Navlinks>
```
Comment (ThomasKirst) [inline pages/components/Footer.js]: I would use the singular here: `NavLink` (and camel case)

File: pages/components/Footer.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        </Link>
+        <Link href="/shoppingcart">
+          <Navlinks>CART</Navlinks>
+        </Link>
+      </Navbar>
+    </NavFooter>
+  );
+}
+
+const Navlinks = styled.a`
```
Comment (ThomasKirst) [inline pages/components/Footer.js]: See my comment above –> `NavLink`

File: pages/components/Layout.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import Footer from './Footer';
+
+export default function Layout({ children }) {
+  return (
+    <>
```
Comment (ThomasKirst) [inline pages/components/Layout.js:6]: `Header` is missing

File: pages/shoppingcart.js [hunk 7]
```diff
@@ -1,13 +1,28 @@
...
   const selection = ingredients.filter((ingredient) => ingredient.chosen);
 
+  function removeBagle(id) {
+    const updatedIngredients = ingredients.map((ingredient) => {
+      if (ingredient.id === id) {
+        ingredient.chosen = false;
+      }
+      return ingredient;
+    });
+    onUpdateIngredients(updatedIngredients);
```
Comment (ThomasKirst) [inline pages/shoppingcart.js:11]: Top 👍

**Review by ThomasKirst — CHANGES_REQUESTED** [review]
Looks really good so far!
I have only a few comments, especially regarding your `Layout` component.
It´s not much and easy to fix ;)

File: pages/about.js (hunk 3 above)
Comment (MerleMak) [inline pages/about.js]:
```suggestion
      <h1>This page was made with a lot of love</h1>
```
😄

**Review by MerleMak — APPROVED** [review]
Looks great! :)
Keep it up!

**Review by ThomasKirst — APPROVED** [review]
Looks good!
Check your Vercel deployment (Build Output Log) before the merge.
The deployment on this branch didn´t seem to work.

**Pull request merged**